sharktank/sharktank/tools/upload_all_models_to_azure.py

```diff
@@ -0,0 +1,51 @@
+from ..utils.azure import upload_all_models
```

Comment:
copyright headers at the tops of files please

```suggestion
# Copyright 2024 Advanced Micro Devices, Inc.
#
# Licensed under the Apache License v2.0 with LLVM Exceptions.
# See https://llvm.org/LICENSE.txt for license information.
# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

from ..utils.azure import upload_all_models
```

Reply:
Done